.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: Use Node 12
```

Comment:
```suggestion
      - name: Use Node 16
```

Reply:
Should we maybe test with both 14 and 16?

Reply:
yup